code/game/objects/structures/signs/signs_warning.dm

```diff
@@ -1,122 +1,181 @@
 
 /obj/structure/sign/warning/fire
-	name = "\improper DANGER: FIRE"
+	name = "\improper DANGER: FIRE sign"
+	sign_change_name = "Warning - Fire Hazard"
 	desc = "A warning sign which reads 'DANGER: FIRE'."
 	icon_state = "fire"
 	resistance_flags = FIRE_PROOF
+	is_editable = TRUE
 
 /obj/structure/sign/warning/nosmoking
-	name = "\improper NO SMOKING"
+	name = "\improper NO SMOKING sign"
+	sign_change_name = "Warning - No Smoking"
 	desc = "A warning sign which reads 'NO SMOKING'."
 	icon_state = "nosmoking2"
 	resistance_flags = FLAMMABLE
+	is_editable = TRUE
 
 /obj/structure/sign/warning/nosmoking/circle
+	name = "\improper NO SMOKING sign"
+	sign_change_name = "Warning - No Smoking Alt"
+	desc = "A warning sign which reads 'NO SMOKING'."
 	icon_state = "nosmoking"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/radiation
-	name = "\improper HAZARDOUS RADIATION"
+	name = "\improper HAZARDOUS RADIATION sign"
+	sign_change_name = "Warning - Radiation"
 	desc = "A warning sign alerting the user of potential radiation hazards."
 	icon_state = "radiation"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/radiation/rad_area
-	name = "\improper RADIOACTIVE AREA"
+	name = "\improper RADIOACTIVE AREA sign"
+	sign_change_name = "Warning - Radioactive Area"
 	desc = "A warning sign which reads 'RADIOACTIVE AREA'."
+	is_editable = TRUE
 
 /obj/structure/sign/warning/xeno_mining
-	name = "\improper DANGEROUS ALIEN LIFE"
+	name = "\improper DANGEROUS ALIEN LIFE sign"
+	sign_change_name = "Warning - Xenos"
 	desc = "A sign that warns would-be travellers of hostile alien life in the vicinity."
 	icon = 'icons/obj/mining.dmi'
 	icon_state = "xeno_warning"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/enginesafety
-	name = "\improper ENGINEERING SAFETY"
+	name = "\improper ENGINEERING SAFETY sign"
+	sign_change_name = "Warning - Engineering Safety Protocols"
 	desc = "A sign detailing the various safety protocols when working on-site to ensure a safe shift."
 	icon_state = "safety"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/explosives
-	name = "\improper HIGH EXPLOSIVES"
+	name = "\improper HIGH EXPLOSIVES sign"
+	sign_change_name = "Warning - Explosives"
 	desc = "A warning sign which reads 'HIGH EXPLOSIVES'."
 	icon_state = "explosives"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/explosives/alt
-	name = "\improper HIGH EXPLOSIVES"
+	name = "\improper HIGH EXPLOSIVES sign"
+	sign_change_name = "Warning - Explosives Alt"
 	desc = "A warning sign which reads 'HIGH EXPLOSIVES'."
 	icon_state = "explosives2"
+	is_editable = TRUE
 
 /obj/structure/sign/warning/testchamber
-	name = "\improper TESTING AREA"
-	desc = "A sign that warns of high-power testing equipment in the area. That's either a really powerful laser... or a satellite landing on some person's head."
+	name = "\improper TESTING AREA sign"
+	sign_change_name = "Warning - Testing Area"
+	desc = "A sign that warns of high-power testing equipment in the area."
```

Comment:
Aww, why'd you remove the soul from this sign's description?

Reply:
This is primarily used for rooms with the BEPIS and experimenter in it, now. It seemed like outdated flavortext, but I suppose it could be given new flavortext.